src/screens/reminder/ReminderInfoScreen.tsx

```diff
@@ -0,0 +1,166 @@
+  startingDateTime: new Date(2023, 2, 31, 13),
+  isRemindCaretaker: true,
+  note: 'mock note',
+  importanceLevel: 'Low',
+  recursion: 'EVERY_DAY',
+  uid: 75,
+  imageid: 'asd15bsads'
+};
+
+const ReminderInfoScreen = ({ route }) => {
+  const { t } = useTranslation();
+  const { language } = useLanguage();
+  const { user } = useContext(UserContext);
+
+  const navigation =
+    useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+
+  const reminderId = useMemo<ReminderNoti>(
+    () => route?.params.info.rid,
+    [route]
+  );
+
+  const [reminderInfo, setReminder] = useState<Reminder>({
+    ...mockReminder,
+    imageid: user?.imageid ?? ''
+  });
+
+  const reminderStatus = useMemo<ReminderStatus>(() => {
+    if (route?.params.info.isDone) return ReminderStatus.DONE;
+    if (Date.now() > reminderInfo.startingDateTime.getTime())
+      return ReminderStatus.OVERDUE;
+    return ReminderStatus.PENDING;
+  }, [route, reminderInfo]);
+
+  const getRecursionLevel = useCallback(() => {
+    const { recursion, customRecursion } = reminderInfo;
+
+    if (recursion) return t(ReminderInfoScreenRecursion[recursion]);
+    if (customRecursion) return t('reminder.customRecursion');
+    return t('reminder.noRecursion');
+  }, [reminderInfo]);
+
+  useAsyncEffect(async () => {
+    const reminderData = (
+      await client.get<Reminder>('/reminder/get/elderly', {
+        params: {
+          rid: reminderId
+        }
+      })
+    ).data;
+
+    setReminder(reminderData);
+  }, []);
+
+  const editReminder = useCallback(
+    () =>
+      navigation.navigate('EditReminderScreen', {
+        info: {
+          dateTime: new Date(),
+          image: undefined,
+          note: 'hi',
+          notifyMyCaretaker: false,
+          repetition: 'everyday',
+          title: 'hello'
+        }
+      }),
+    []
+  );
+
+  return (
+    <View px={5}>
+      <ReminderStatusBar status={reminderStatus} />
+      <Spacer />
+      <HStack justifyContent="space-between">
+        <FormHeader headerText={reminderInfo?.title ?? ''} />
+        <Button
+          background="muted.200"
+          onPress={editReminder}
+          leftIcon={
+            <Icon
+              ml="-1"
+              as={MaterialCommunityIcons}
+              name="pencil"
+              size="5"
+              color="muted.600"
+            />
+          }>
+          <Text color="muted.600">{t('reminder.edit')}</Text>
+        </Button>
+      </HStack>
+      <HStack my={2}>
+        <Box
+          backgroundColor={statusDecoration[reminderStatus].chip}
+          p={1}
+          mr={2}
+          borderRadius={2}>
+          {moment(reminderInfo?.startingDateTime).format('DD/MM/YYYY')}
+        </Box>
+        <Box
+          backgroundColor={statusDecoration[reminderStatus].chip}
+          p={1}
+          borderRadius={2}>
+          {moment(reminderInfo?.startingDateTime).format(
+            `HH:MM ${language() === 'en' ? 'A' : 'น.'}`
+          )}
```

Comment:
what is 'A' here?

Reply:
This is a convenient formatting which becomes AM or PM depending on the time. It only provides English text, so I added น. at the back @pointmekin